Tasks/Xcode/xcodeutils.ts

```diff
@@ -1,4 +1,13 @@
 import tl = require('vsts-task-lib/task');
+const readline = require('readline');
```

Comment:
Add a test for reading the provisioning style. This will probably require switching over from 'readline' and 'fs' to something with mocking hooks.

Reply:
calls to fs can be mocked in the new test framework, I have an example of that in the "Install Apple Certificate" task tests. I was hoping to do that change for a while to avoid created files on disk during L0 test runs.